app/views/declaration/package_information.scala.html

```diff
@@ -42,7 +43,7 @@
     title = Title("supplementary.packageInformation.title", "supplementary.items")
 ) {
 
-    @components.back_link(Navigator.backLinkItemPage(PackageInformation)(request)(mode, itemId), mode)
+    @components.back_link(Navigator.backLink(PackageInformation, mode, ItemId(itemId)))
```

Comment:
You could pass ItemId into the view

And maybe add it to the model so the routes with an item-id are type safe :D